### ISSUE TITLE
It's not plugged in yet.

### ISSUE THREAD

**Issue by pwillard** [FrannDzs/Route-Riter#1]
It's still more "dependency hell" than "jump in and start coding" in its current state.

**Comment by FrannDzs** [comment]
> It's still more "dependency hell" than "jump in and start coding" in its current state. But It might be my fault for trying to use my Windows XP Virtual Machine that already had VB6 installed.

I know the guide may not be clear. I would like some feedback from someone else on this. Try reinstalling everything following the steps. Then let me know what happened, so I can somehow fix the guide if that is the problem.

Were you able to load the project at least?
 Thanks!

**Comment by FrannDzs** [comment]
The VSFLEX controls I updated may not be compatible with WINXP although I am not completely sure. Maybe the installer is not compatible, then I can directly provide the .ocx files.

**Comment by pwillard** [comment]
Yup, that must be it... XP is too old.  The components won't install.

**Comment by pwillard** [comment]
I can load it... but even after OCX installs...  the components are not there so VB6 is unhappy.  I'll try it on a different virtual machine.  I *will not* put VB6 on my main PC... for multiple *reasons*.  

So, for everyone watching... don't use XP 32bit.

**Comment by FrannDzs** [comment]
> 

I will add the .ocx 32 bit controls so you can install it manually

**Comment by FrannDzs** [comment]
> I can load it... but even after OCX installs... the components are not there so VB6 is unhappy. I'll try it on a different virtual machine. I _will not_ put VB6 on my main PC... for multiple _reasons_.
> 
> So, for everyone watching... don't use XP 32bit.

I have added the cabs files containing the C1 controls. Try it and let me know what happened.

https://github.com/FrannDzs/Route-Riter/tree/main-(7.6.27)/Source/Dependancies/C1%20ActiveX%20Controls/ActiveX%20cabs

Remember you only need c1sizer.ocx, vsflex8l.ocx and vsprint8.ocx

**Comment by FrannDzs** [comment]
I have the controls working with the latest versions on windows 10. I look forward to your response with the older versions.

![Captura de pantalla 2022-05-20 143042](https://user-images.githubusercontent.com/76663897/169581832-bffaea14-0bde-4e01-8f74-9b3731f2f174.png)
![Captura de pantalla 2022-05-20 143225](https://user-images.githubusercontent.com/76663897/169581861-30e9b69e-69ec-4bfa-9af4-91ef45b6e770.png)

**Comment by FrannDzs** [comment]
> Yup, that must be it... XP is too old. The components won't install.

Were you able to load the controls?

The updated C1 controls are only compatible from Windows 7 onwards.
